jpsonic-main/src/main/java/com/tesshu/jpsonic/service/upnp/processor/ArtistByFolderUpnpProcessor.java

```diff
@@ -144,10 +129,9 @@ protected static final String createMusicFolderId(String id) {
     @Override
     public List<FolderArtistAlbumWrapper> getChildren(FolderArtistAlbumWrapper item, long first, long maxResults) {
         if (item.isArtist()) {
-            return getDispatcher()
-                    .getAlbumProcessor().getAlbumsForArtist(item.getName(), first, maxResults,
-                            util.isSortAlbumsByYear(item.getName()), util.getGuestMusicFolders())
-                    .stream().map(FolderContent::new).collect(toList());
+            return albumDao.getAlbumsForArtist(first, maxResults, item.getName(),
+                    util.isSortAlbumsByYear(item.getName()), util.getGuestFolders()).stream().map(FolderContent::new)
+                    .collect(toList());
```

Comment:
No need to use Collectors anymore.

Reply:
I'm certainly curious, but there were 73 locations, so I'll probably be able to rewrite it after closing this pull request.

Reply:
This pull request will only target upnp package and below.